### DISCUSSION TITLE
The symbol for adding and removing an issue to the epic is the same.

### DISCUSSION THREAD

**Discussion by rajatagarwal-ibm** [community/community#125009] [Projects and Issues]
### Body

The symbol for adding and removing an issue to the epic is the same. Could be different for the better understanding. 
![Screenshot 2024-05-21 at 13 16 48](https://github.com/community/community/assets/108140212/6f93a7c1-b3e9-41cf-9e55-b52533bea858)

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @rajatagarwal-ibm , we're happy you're here! You are more likely to get a useful response if you are posting in the applicable category and are explicit about what your feedback entails. I've gone ahead and moved it for you. Good luck!
